web/public/themes/noir-gold.css

```
@@ -113,9 +113,9 @@ html[data-theme="noir-gold"] {
   --blue-bg: var(--accent-bg);
 
   --font-sans:
-    "Source Serif 4", "IBM Plex Serif", Georgia, "Times New Roman", serif;
-  --font-serif: "Fraunces", Georgia, "Times New Roman", serif;
-  --font-logo: "Fraunces", Georgia, "Times New Roman", serif;
+    "Geist Sans", -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.11.1)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)